code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -359,6 +359,17 @@ var/list/tag_suits_list = list()
 	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank/emergency_oxygen,/obj/item/weapon/tank/emergency_nitrogen,/obj/item/toy)
 	body_parts_covered = ARMS|LEGS|FULL_TORSO
 
+/obj/item/clothing/suit/timefake
+	name = "time suit"
+	desc = "A replica of a real time suit, it helps you remember the previous timeline, may the loop protect their souls."
```

Comment:
Can't decide which of the many memes about there being two of something to use but there are two comma splices in here

Reply:
not a good idea to write your "lore" when tired i tell you that

Reply:
Gee Bill was my first thought but I didn't want to either type the whole thing out or risk being called out for half-assing it